styles.css

```diff
@@ -0,0 +1,157 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+@media all and (min-width: 375px) {
+  body {
+    background-image: url(header-illsutration-mobile@2x.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position: 0 3em;
+  }
+
+  header {
+    gap: 100px;
+  }
+
+  .appbar {
+    display: flex;
+    flex: 1;
+    width: 100%;
+    height: 20px;
+    padding: 10px;
+    padding-bottom: 0;
+    box-sizing: border-box;
+  }
+
+  .times {
+    flex: 10;
+  }
+
+  .group3 {
+    flex: 1;
+  }
+
+  .nav {
+    display: none;
+  }
+
+  .bar {
+    padding: 10px;
+    padding-top: 20px;
+    position: absolute;
+    right: 1.2em;
+    font-size: 1.3em !important;
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 0;
+    padding-left: 40px;
+    padding-right: 40px;
+    padding-bottom: 40px;
+    position: relative;
+    top: 5em;
```

Comment:
These are not actually needed to use here. Instead of using position top property you should use margin-top to adjust the content to the center both in mobile and desktop accordingly